### PR TITLE
Prepare changelog for 0.32.0

### PR DESCRIPTION
**Pull request by brusherru**
It's a part of releasing pipeline.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+* [core] Add `=XOD_PROJECT` literal (#1889)
+* [core] Add `=XOD_TOKEN` literal (#1900)
+* [ide] Derive project name from the saved filename in desktop IDE (#1904)
+* [ide] Switch to another compilation limits backend (#1908)
+* [ide] Add search for libraries within XOD IDE (#1909)
+
+### Bug fixes
+
+* [core] Transpile buses in specialization nodes properly to keep the program graph (#1905)
+* [core] Fix arduino-cli and spaces in username on Windows (#1906)
```
Comment (nkrkv) [inline CHANGELOG.md]: Too much tech details. Better:

> Compilation no longer fails on Windows if PC username contains spaces

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## Not yet released
+
+### Features and enhancements
+
+* [core] Add `=XOD_USERNAME` literal (#1888)
+* [core] Add `=XOD_PROJECT` literal (#1889)
+* [core] Add `=XOD_TOKEN` literal (#1900)
```
[the remark this answers is not in the document]
Comment (brusherru) [reply CHANGELOG.md]: @nkrkv I'm not sure about changes to these lines. Check it out, please

[the remark this answers is not in the document]
Comment (nkrkv) [reply CHANGELOG.md]: Sounds nice :ok_hand:

[the remark this answers is not in the document]
Comment (brusherru) [reply CHANGELOG.md]: 🙌

**Pull request merged**